src/core/adapters/officialOnyxiaApiClient.ts

```diff
@@ -251,6 +252,7 @@ export function createOfficialOnyxiaApiClient(params: {
                             const common: DeploymentRegion.S3.Common = {
                                 "monitoringUrlPattern": S3.monitoring?.URLPattern,
                                 "defaultDurationSeconds": S3.defaultDurationSeconds,
+                                "acceptBucketCreation": S3.acceptBucketCreation ?? true,
```

Comment:
- To be grammatically correct it should be `acceptsBucketCreation` (with an s)
- Throughout the codebase all boolean start either by `is`, `does` or `do`.  

I suggest using `isBucketCreationAllowed`.  
Never mind if the API and the internal types diverge.